definitions/tools/rename.cwl

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env cwl-runner
+
```

Comment:
a comment or description mentioning that this doesn't work with cromwell would probably be a good idea, so we don't try to use it in workflows at present.

Reply:
Good point; I've added a `label` and `doc` to both tools.